README.md

```diff
@@ -8,85 +8,18 @@ Wazuh
 
 Wazuh is an Open Source Host-based Intrusion Detection System that performs log analysis, file integrity monitoring, policy monitoring, rootkit detection, real-time alerting, active response, vulnerability detector, etc.
 
-In this repository, you can find the necessary tools to build a Wazuh package for Debian based OS (.deb) and an RPM based OS package (.rpm).
+In this repository, you can find the necessary tools to build a Wazuh package for Debian based OS (.deb), RPM based OS package (.rpm), MacOS (.macos) and AIX (.aix), the OVA, and the apps for Kibana and Splunk:
```

Comment:
The file extensions are incorrect and necessaries.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/162/commits/a03ea419f6431886c0f5b99fdb607af85436ba36